### PR TITLE
Improve logging and add an option to run 3play translations.

### PR DESCRIPTION
**Pull request by Qubad786**
[EDUCATOR-1979](https://openedx.atlassian.net/browse/EDUCATOR-1979)

Improve logging and add an option to run 3play translations with heal.

### CODE REVIEW

**Comment by Qubad786** [comment]
@mushtaqak @muzaffaryousaf quick review?

File: bin/heal [hunk 1]
```diff
@@ -71,12 +72,27 @@ def main():
...
 
     veda_id = args.veda_id
     course_id = args.course_id
     schedule = args.schedule
 
+    process_translations = args.process_translations
+    if process_translations:
+        # Only kick off a round of retrieving successful
+        # translations from 3Play Media
+        os.environ['DJANGO_SETTINGS_MODULE'] = 'VEDA.settings.production'
```
Comment (mushtaqak) [inline bin/heal]: Do we need to add this here ?

Comment (Qubad786) [reply bin/heal]: yes. it is to tell Django about the correct settings.

Comment (Qubad786) [reply bin/heal]: This is temporary and will be removed once we have Django mgmt. command for this.

**Comment by Qubad786** [comment]
Going to merge.

**Pull request merged**